ganga/GangaLHCb/Lib/LHCbDataset/BKQuery.py

```diff
@@ -11,7 +11,10 @@
 from GangaCore.Utility.logging import getLogger
 logger = getLogger()
 #\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\#
+from GangaCore.Utility.Config import getConfig
 
+configLHCb = getConfig('LHCb')
+defaultCompressed = configLHCb['compressedDataset']
```

Comment:
Is there any reason for making this configurable?

Reply:
I figured it would be nice for the user to not have to select `True` by default. But maybe it does not matter. I think I will make the compressed option the standard for BK query and get rid of the configurable.